src/main/java/xbot/common/subsystems/drive/SwerveSimpleTrajectoryMode.java

```diff
@@ -0,0 +1,9 @@
+package xbot.common.subsystems.drive;
+
+/* Different ways to render the SwerveSimpleTrajectoryCommand's path */
+public enum SwerveSimpleTrajectoryMode {
```

Comment:
⭐ could you add more comments about what each mode does? What would you tell a new student that had to do use this about their choices?